### PR TITLE
Release/0106

### PR DESCRIPTION
**Pull request by johndendron**
## What issues does this PR fix or reference?

- Fixes:
OR
- Relates to:

> **NOTE:** `Fixes` will close the ticket after merging. `Relates to:` will leave the ticket open after merging.

### Merge requirements satisfied

> For more information, visit the [Contributing Guide for Documentation](https://wiki.dendron.so/notes/b58801fc-43a9-4d42-a58b-eabc3e8538cb/)

- [ ] Check rendered output to ensure formatting is correct for renderings of wikilinks, note refs, tables, and images. `Dendron: Show Preview` can be used in your workspace to confirm the page renders as expected. Sometimes, `Dendron: Reload Index` needs to be ran if certain wikilinks aren't working as expected. (ignore this if contribution is made via the `Edit this page on GitHub` button from the published site)
- [ ] If this PR includes any refactoring (renaming notes, renaming headers, etc.), make sure that these changes were done via [Dendron Refactoring Commands](https://wiki.dendron.so/notes/srajljj10V2dl19nCSFiC/). This will ensure that any other impacted areas of the documentation, that link to or embed modified notes (via note references), are automatically updated.

Verify GitHub Actions tests are passing, which can be seen in the `Checks` tab of the PR:

- [ ] `URL validator` GitHub Action passes
- [ ] `Dendron site build` GitHub Action passes

> **NOTE:** If you are a new contributor, a maintainer will need to provide approval for GitHub Actions to run on your PRs.

### CODE REVIEW

**Comment by Harshita-mindfire** [comment]
Adding a comment here for visiblity: 
new addition in changelog: `fix(views): remove forced tree view focus on workspace activation #3313 @hikchoi`

File: vault/changelog.md [hunk 1]
```diff
@@ -2,10 +2,31 @@
...
+
+- the `Delete Node` command is now replaced with the `Delete` command (if you haven't set any custom keyboard shortcuts for `Delete Node`, no change is necessary)
+- the `Show Preview` command is now replaced with the `Toggle Preview` command (if you haven't set any custom keyboard shortcuts for `Show Preview`, no change is necessary)
+
+### Enhancements
+
+- enhance(edit): remove delete node command (#3285) @kevin
+- enhance(retrieve): remove `Show Preview` command (#3276) @kevin
+- enhance(workspace): `Fix It` button in warning toaster for duplicate note id (#3237) @joshi
+- enhance(retrieve): fix default keyboard shortcut of toggle preview on windows (#3276) @kevin
```
Comment (Harshita-mindfire) [inline vault/changelog.md:21]: `enhance(retrieve): fix default keyboard shortcut of toggle preview on windows (#3276) @kevin` 
can be updated to : 
`- fix(retrieve): windows keybindings for toggle preview(#3323) @kevin`

File: vault/changelog.release.2022-08-02.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+title: 0.106
+desc: ''
+updated: 1659450027598
+created: 1659372767472
+---
+
+Dendron 0.106 has sprouted  🌱
+August 02, 2022
+
+**Remove Delete Node and Show Preview commands** The `Delete Node` command is now replaced with the `Delete` command and `Show Preview` command is now replaced with the `Toggle Preview` command. Time to change your custom keyboard shortcuts to reflect the changes!
```
Comment (kevinslin) [inline vault/changelog.release.2022-08-02.md:12]: If you've set custom keyboard shortcuts for these commands, a reminder to change them now!

File: vault/changelog.release.2022-08-02.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## Highlights
+
+- enhance(edit): remove delete node command
```
Comment (kevinslin) [inline vault/changelog.release.2022-08-02.md:17]: remove `Delete Node` command

**Review by kevinslin — CHANGES_REQUESTED** [review]

**Review by kevinslin — APPROVED** [review]

**Pull request merged**